tests/k6/configs/k8s/base/k6.yaml

```diff
@@ -35,6 +35,14 @@ spec:
         #   "csv=results/base.gz",
         #   "scenarios/infer_constant_rate.js",
         #   ]
+        # # k8s-test-script
```

Comment:
The idea here would be that atm we'll manually comment/uncomment depending on what types of tests we want to run right?